src/styles/styles.scss

```
@@ -2,6 +2,10 @@
 :root {
   --primary_color: #5b78f6;
   --dark: #323232;
+  --font-sans: 'Google Sans Text', 'Lato', system-ui, -apple-system,
+    BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)